src/reference-manual/pedantic-mode.Rmd

```diff
@@ -0,0 +1,363 @@
+
+
+# Pedantic Mode
+
+Pedantic Mode is a compilation option built into Stanc3 that warns you about potential issues in your Stan program.
+
+For example, if you compile the following program with Pedantic Mode:
```

Comment:
As @bbbales2 says, these are sentence fragments.  Grammatically, a dislayed formula is a noun and a displayed equation is like an embedded clause (and we can use clauses like nouns in the way we can use statements like expressions in programming languages, so it gets complicated).   

The way to fix this is to use something like

```
Consider compiling the following program with pedantic mode.

blah-blah blah

This program provides the following output.

blah-blah-blah

Reply:
I went with:

> For example, consider the following program.
> 
> blah-blah blah
> 
> Pedantic mode produces the following warning.
> 
> blah-blah-blah

Reply:
blah-blah-blah approved!